backend/migrations/20240524154034_incident_data.sql

```diff
@@ -0,0 +1,20 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE incident_data (
+    instance_id VARCHAR(255) NOT NULL,
+    incident_id VARCHAR(255) NOT NULL,
+    title VARCHAR(255) NOT NULL,
+    description VARCHAR(255) NOT NULL,
+    timestamp TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    severity SMALLINT NOT NULL,
+    status SMALLINT NOT NULL,
+    created_by VARCHAR(255) NOT NULL
```

Comment:
Should at another `created_at` column, since it might be a different time than the `timestamp` column

Reply:
so timestamp will record when incident occurs and created_at is when it is reported?